scripts/mixins/rotz_bodyguarded_nm.lua

```diff
@@ -0,0 +1,77 @@
+-- Mixin for behavior that is common to five RoTZ NMs that have 21-24 hour respawns
+-- These mobs are Centurio XII-I, Meteormauler Zhagtegg, Coo Keja the Unseen, Meww the Turtlerider, and Bright-handed Kunberry
+-- These NMs have two bodyguard mobs that spawn with them if beastmen control the region they spawn in
+-- Also they have 50% damage taken down mods
+require('scripts/globals/mixins')
+
+g_mixins = g_mixins or {}
+
+g_mixins.bodyguard = function(bodyguardedNM)
+    bodyguardedNM:addListener('SPAWN', 'ROTZ_BODYGUARDED_NM_SPAWN', function(mob)
+        -- retail captures show these mods are not dependent on region control
+        mob:setMod(xi.mod.UDMGPHYS, -5000)
+        mob:setMod(xi.mod.UDMGRANGE, -5000)
+        mob:setMod(xi.mod.UDMGBREATH, -5000)
+        mob:setMod(xi.mod.UDMGMAGIC, -5000)
```

Comment:
2 things:
1. This are literally all damage global types. Just use the 1 modifier for them all: `xi.mod.DMG`
2. Even if all of this mobs happen to share the -50% damage taken, this mixing should stick to the bodyguard logic. What this means is this modifier should be placed in themob script

Reply:
> 2 things:
> 
> 1. This are literally all damage global types. Just use the 1 modifier for them all: `xi.mod.DMG`
> 2. Even if all of this mobs happen to share the -50% damage taken, this mixing should stick to the bodyguard logic. What this means is this modifier should be placed in themob script

1. Yeah I thought about using DMG but that is capped at -50% so if the mob also uses some DMG or DMGPHY modifying buff then I think it would have no impact as DMG would already be at the cap.
2. Yep, thanks for feedback I have updated the PR!

Reply:
Just to note here as well the Centurio X-III changes and bodyguard behavior was moved to a separate PR for ease of review and better separation of PRs. I will link to this comment in that PR.